lib/msal-react/docs/class-components.md

```diff
@@ -0,0 +1,188 @@
+# Using msal-react with class components
+
+`msal-react` does support both function components and class components. However, you will not be able to use `msal-react` hooks inside your class components so if you need access to authentication state inside your class component you will need to use `msal-browser` directly to obtain similar functionality.
+
+## Initialization
+
+Just like when using function components you will need an `MsalProvider` component at the top level of the component tree that requires access to authentication state.
```

Comment:
```suggestion
Just like when using function components you will need a `MsalProvider` component at the top level of the component tree that requires access to authentication state.
```

Reply:
I don't think this is right, would love to be educated though haha

Reply:
Oh, I think you're right, actually. I forgot to read it out loud. Sorry!